### PR TITLE
Model imports

### PR DESCRIPTION
**Pull request by Abbondanzo**
Fixes imports of model types, moving everything else to Typescript declaration files. Will make it easier to interact with model types in the future.

### CODE REVIEW

**Comment by Abbondanzo** [comment]
Ignore the line count--I reset the lock file because it wasn't working cross platform

**Comment by Abbondanzo** [comment]
Will revert linting rules momentarily

File: webapp/src/Components/Replay/BasicStats/PlayerStats/PlayerStatsTabs.tsx [hunk 1]
```diff
@@ -43,13 +43,14 @@ class PlayerStatsTabsComponent extends React.PureComponent<Props> {
...
-                {playerStatsSubcategoryValues
-                    .map((subcategory: PlayerStatsSubcategory) =>
-                        <Tab label={subcategory} value={subcategory} key={subcategory}
-                             icon={<FontAwesomeIcon icon={categoryToIcon[subcategory]}/>}
+                {Object.keys(PlayerStatsSubcategory).map((subcategory: PlayerStatsSubcategory) => {
+                    const value = PlayerStatsSubcategory[subcategory]
+                    return <Tab label={value}
+                                value={value}
+                                key={value}
+                                icon={<FontAwesomeIcon icon={categoryToIcon[value]} />}
```
Comment (twobackfromtheend) [inline webapp/src/Components/Replay/BasicStats/PlayerStats/PlayerStatsTabs.tsx]: As far as I can tell, we aren't going with the space (at least not yet?)

Comment (Abbondanzo) [reply webapp/src/Components/Replay/BasicStats/PlayerStats/PlayerStatsTabs.tsx]: Force of habit. I've fixed it

**Pull request merged**